src/System.Private.CoreLib/shared/System/Delegate.cs

```diff
@@ -0,0 +1,107 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.Serialization;
+
+namespace System
+{
+    public abstract partial class Delegate : ICloneable, ISerializable
+    {
+		internal Delegate()
```

Comment:
Nit: this file appears to be using tabs in many places... can you change it to use spaces instead?